code/modules/mob/living/carbon/human/human.dm

```diff
@@ -215,6 +215,13 @@
 	if(istype(MB))
 		MB.RunOver(src)
 
+	if(ishuman(AM) && lying)

```

Comment:
But I am asking that Dwarves are entirely UNABLE to trample someone, as they are too small. I ask again; How is this "buffing manlet power even more"?